packages/amplication-build-manager/src/build-runner/build-runner.controller.spec.ts

```diff
@@ -191,7 +192,7 @@ describe("BuildRunnerController", () => {
     await controller.onCodeGenerationFailure(codeGenerationFailureDTOMock);
 
     expect(mockKafkaServiceEmitMessage).toBeCalledWith(
-      configService.get(Env.CODE_GENERATION_FAILURE_TOPIC),
+      configService.get(KAFKA_TOPICS.CODE_GENERATION_FAILURE_TOPIC),
```

Comment:
Same as https://github.com/amplication/amplication/pull/6698/files#diff-243e5dc75c8a6ca029dac34dadf96afd121d56b8c6895888240190d8a1a3f4d2R24
```suggestion
      KAFKA_TOPICS.CODE_GENERATION_FAILURE_TOPIC,
```

Reply:
@barshimi ?